style.css

```diff
@@ -1,3 +1,10 @@
 body {
-	
-}
\ No newline at end of file
+background-color: grey;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step/pullRequest?prid=5896505)